docs/administration.platforms.md

```diff
@@ -0,0 +1,12 @@
+---
+id: administration.platforms
+title: Platform Notifications
+---
+
+Administrators can inform users about platform specific events by creating platform notifications:
```

Comment:
d: Mention some use cases on when a platform notification might make sense. Platform outages or issues, planned maintenance windows, etc. Relating to these examples you can also explain which severity level fits best for whoch example.